src/styles/styles.css

```diff
@@ -126,4 +126,64 @@ div.row2 .modalBtn:visited {
   .UserTable .FieldTitle {
     width: 100%;
   }
-}
\ No newline at end of file
+}
+.container-custom {
```

Comment:
diverse screen sizes included looks good. good job on taking the time to do that....
Is there a reason we have double media queries... not an attack just wondering as a learner:
@media (min-width: 476px) {
  .container-custom {
      padding-right: 15px;
      padding-left: 15px;
  }

AND 

@media (min-width: 476px) {
  .container-custom {
      width: 100%;
  }